debian/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM        --platform=linux/amd64 debian:buster-slim
+
+LABEL           author="Michael Parker" maintainer="parker@pterodactyl.io"
+
+LABEL       org.opencontainers.image.source="https://github.com/pterodactyl/yolks"
+LABEL       org.opencontainers.image.licenses=MIT
+
+RUN         apt update \
```

Comment:
This seems like more run layers than necessary, they can all be merged into a single layer.